source/localizable/getting-started/quick-start.md

```diff
@@ -256,4 +256,12 @@ everyone relies on. If you're interested in deploying your app to
 production in a fast and reliable way, check out the [Ember CLI
 Deploy][ember-deploy] addon.
 
-[ember-deploy]: http://ember-cli.github.io/ember-cli-deploy/
+[ember-deploy]: http://ember-cli-deploy.github.io/ember-cli-deploy/
+
+If you deploy your application to an Apache web server, first create a 
```

Comment:
Can you insert a newline after each comma as per our [style guide](https://github.com/emberjs/guides/blob/master/CONTRIBUTING.md#style-guide)? Thanks :)